app/src/main/res/layout/activity_welcome.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/colorPrimary">
+
+    <!-- activity_screen_slide.xml -->
+    <androidx.viewpager2.widget.ViewPager2
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:id="@+id/pager"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent" />
+
+    <Button
+        android:id="@+id/next_button"
+        android:text="NEXT"
```

Comment:
text should be extracted into string file.

Reply:
Also, you might want to make the text in `strings.xml` be "Next" and then transform it to uppercase in the layout file. That way we can easily flip it to normal case. Since we're only in English now, there's not really an apparent reason to go one way or the other, but if we had support for additional languages, we might not know how to convert them back from uppercase ourselves.